plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -1578,7 +1578,7 @@ public ConnectorTableHandle beginUpdate(ConnectorSession session, ConnectorTable
         Table table = metastore.getTable(identity, tableName.getSchemaName(), tableName.getTableName())
                 .orElseThrow(() -> new TableNotFoundException(tableName));
 
-        if (!isTransactionalTable(table.getParameters())) {
+        if (!isFullAcidTable(table.getParameters())) {
             throw new TrinoException(NOT_SUPPORTED, "Hive update is only supported for transactional tables");
```

Comment:
Squash this into the commit that updates the message, since those changes are coupled